gobin/exe.go

```diff
@@ -70,15 +75,50 @@ func toPackages(ctx context.Context, out *[]*claircore.Package, p string, r io.R
 
 	ev := zlog.Debug(ctx)
 	vs := map[string]string{
-		"runtime": bi.GoVersion,
+		"stdlib": bi.GoVersion,
 	}
 	var mainVer claircore.Version
+	var mmv string
 	mpv, err := semver.NewVersion(bi.Main.Version)
 	switch {
 	case errors.Is(err, nil):
 		mainVer = fromSemver(mpv)
+	case bi.Main.Version == `(devel)`:
+		// This is currently the state of any main module built from source; see
+		// the package documentation. Don't record it as a "bad" version and
+		// pull out any vcs metadata that's been stamped in.
+		mmv = bi.Main.Version
+		var v []string
+		for _, s := range bi.Settings {
+			switch s.Key {
+			case "vcs":
+				v = append(v, s.Value)
+			case "vcs.revision":
+				switch len(s.Value) {
+				case 40, 64:
+					v = append(v, "commit "+s.Value)
+				default:
+					v = append(v, "rev "+s.Value)
+				}
+			case "vcs.time":
+				v = append(v, "built at "+s.Value)
+			case "vcs.modified":
+				switch s.Value {
+				case "true":
+					v = append(v, "dirty")
+				case "false":
+				default:
+					panic("unreachable")
```

Comment:
Do we want to risk the, however tiny, chance that someone is doing something messed up with their compiler and is able to down an indexer?

Reply:
Good point, easy change.